### ISSUE TITLE
Double volume flow at mesh boundary

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#10717]
Looks like we are double counting AREA INTEGRAL at a mesh boundary.  The device on the mesh boundary reports twice the volume flow as the device inside the second mesh.

```
&HEAD CHID = 'Test' /

&MESH IJK=10,10,10, XB=0,2,0,2,0,2/
&MESH IJK=10,10,10, XB=2,4,0,2,0,2/

&TIME T_END = 60 /

&VENT SURF_ID='OPEN' XB=4,4,0,2,0,2/
&VENT SURF_ID='OPEN' XB=-2,-2,0,2,0,1/

&VENT XB=0,0,0,2,0,2, SURF_ID='BLOWER'/
&SURF ID='BLOWER', VOLUME_FLOW = -1, COLOR='GREEN YELLOW', TAU_V=-0.1 /

&DEVC QUANTITY='U-VELOCITY', XB=2.0,2.0,0,2,0,2, ID='VOL_FLOW_MESH_BOUND', SPATIAL_STATISTIC='AREA INTEGRAL'/
&DEVC QUANTITY='U-VELOCITY', XB=2.2,2.2,0,2,0,2, ID='VOL_FLOW', SPATIAL_STATISTIC='AREA INTEGRAL'/

&DEVC QUANTITY='NORMAL VELOCITY', XB=0,0,0,2,0,1, ID='VOL_FLOW_BLOWER_up'  , SPATIAL_STATISTIC='SURFACE INTEGRAL'/
&DEVC QUANTITY='NORMAL VELOCITY', XB=0,0,0,2,1,2, ID='VOL_FLOW_BLOWER_down', SPATIAL_STATISTIC='SURFACE INTEGRAL'/

&TAIL /
```

**Comment by rmcdermo** [comment]
This should be fixed with PR #10720 .  I'll let firebot crunch on it tonight before closing.

**Comment by rmcdermo** [comment]
I think we can close this now.
